src/molecules/SongList.js

```diff
@@ -2,9 +2,9 @@ import React from 'react'
 import { Composition } from 'atomic-layout'
 import Song from './Song'
 
-const SongList = props => (
+const SongList = ({ list }) => (
   <Composition gap="6px">
-    {props.list.slice(0, 5).map(song => {
+    {list.slice(0, 5).map(song => {
```

Comment:
Don't forget to solve the problem that right now all songs are sliced to 5 ;) 

a) Make the slicing length parametric (via props).
b) Or, slice the `list` _before_ passing it to the `<SongList/>`